src/test/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContentTest.java

```diff
@@ -307,7 +289,6 @@ public void shouldSuccessfullyApplyCivilPartnershipContentForNoticeOfProceedings
                 entry(DIVORCE_OR_END_YOUR_CIVIL_PARTNERSHIP, APPLICATION_TO_END_YOUR_CIVIL_PARTNERSHIP),
                 entry(BEEN_MARRIED_OR_ENTERED_INTO_CIVIL_PARTNERSHIP, ENTERED_INTO_A_CIVIL_PARTNERSHIP_WITH),
                 entry(MARRIAGE_OR_CIVIL_PARTNER, CIVIL_PARTNERSHIP),
-                entry("ctscContactDetails", ctscContactDetails),
```

Comment:
As per my previous comment, if you update the TestDataHelper.getBasicDocmosisTemplateContent  method to include ctsc, we can add this back and confirm they are set correctly?